xhr.bs

```diff
@@ -2031,6 +2039,19 @@ attributes initialized to false, so it is suggested that for consistency all
 {{ProgressEvent}} interface do the same.
 
 
+<h3 id=feature-policy>Feature Policy Integration</h3>
+
+<p>This specification defines a policy-controlled feature named <dfn>Synchronous
+XMLHttpRequest</dfn>.
```

Comment:
Are all features supposed to follow this naming convention?

Reply:
I wanted to decouple the name of the feature from the character string used to denote it in policies

The spec just says that there are [policy-controlled features](https://wicg.github.io/feature-policy/#features), and they have a *feature name*, which is defined in the ABNF as `1*( ALPHA / DIGIT / "-")`